eng/native/functions.cmake

```diff
@@ -206,6 +206,27 @@ function(compile_asm)
   set(${COMPILE_ASM_OUTPUT_OBJECTS} ${ASSEMBLED_OBJECTS} PARENT_SCOPE)
 endfunction()
 
+# add_component(componentName [targetName] [EXCLUDE_FROM_ALL])
+function(add_component componentName)
+  if (${ARGC} GREATER 2 OR ${ARGC} EQUAL 2)
```

Comment:
lol there's no `>=` operator?

Reply:
Not in CMake 3.6, which we still need to support for Linux source build.